mars-cli/mars_lib/submit.py

```diff
@@ -59,6 +77,37 @@ def submission(
         f"ISA JSON with investigation '{isa_json.investigation.title}' is valid."
     )
 
+    time_stamp = datetime.timestamp(datetime.now())
+
+    if DEBUG:
+        save_step_to_file(time_stamp, "0_Initial_ISA_JSON_in_model", isa_json)
+
+    if all(
+        repo not in TargetRepository.available_repositories()
+        for repo in target_repositories
+    ):
+        raise ValueError("No target repository selected.")
+
+    if TargetRepository.BIOSAMPLES in target_repositories:
+        # Submit to Biosamples
+        biosamples_result = submit_to_biosamples(
+            isa_json=isa_json,
+            biosamples_credentials=user_credentials,
+            biosamples_url=urls["BIOSAMPLES"]["SUBMISSION"],
+            webin_token_url=urls["WEBIN"]["TOKEN"],
+        )
+        print_and_log(
+            f"Submission to {TargetRepository.BIOSAMPLES} was successful. Result:\n{biosamples_result.json()}",
```

Comment:
In the future I think we could only log the actual response in debug mode, but for now I think it makes sense to keep this in (unless we don't want it for the screencast... easy change to make later though)